src/ts/swagger/index.ts

```diff
@@ -29,6 +33,42 @@ const onWindowLoad = () => {
   };
 
   browser.runtime.onMessage.addListener(messageHandler);
+
+  // Does not seem to need `bind`
+  const setSpec = editor.unmockActions.setSpec;
+  await fillFromTab(setSpec);
+};
+
+const fillFromTab = async (updateSpec: (content: string) => void) => {
+  const result = (await storage.getTabInfo()) || {};
+  const tabIdOrUndefined = result.tabId;
+  if (!tabIdOrUndefined) {
+    console.warn("Cannot fill Swagger editor as no tab ID available");
+    return;
+  }
+  const tabId = tabIdOrUndefined;
```

Comment:
Why reallocation to a new variable? `tabId === tabIdOrUndefined === result.tabId`?

Reply:
Just to make it clear it's not undefined anymore.